components/Settings/ProfileTab.js

```diff
@@ -0,0 +1,32 @@
+import s from 'styled-components'
+
+const Card = s.div`
+  justify-content: space-between;
+  height: 100%;
+  font-family: HelveticaNeue;
+  font-size: 18px;
+`
+
+export default (props) => {
+  const { name, username, email } = props.defaults
+
+  return (
+    <Card>
+      <div className='columns is-mobile'>
```

Comment:
```suggestion
      <div className="columns is-mobile">
```
use double quotes in jsx